portal/server/controllers/utilisation-report-service/utilisation-report-upload/utilisation-report-cell-validators/generate-payment-currency-error.js

```diff
@@ -1,16 +1,28 @@
 const validator = require('validator');
+const { UTILISATION_REPORT_HEADERS } = require('../../../../constants');
 
-const generatePaymentCurrencyError = (paymentCurrencyObject, exporterName) => {
-  if (!paymentCurrencyObject?.value) {
+const generatePaymentCurrencyError = (csvDataRow) => {
+  if (
+    !csvDataRow[UTILISATION_REPORT_HEADERS.PAYMENT_EXCHANGE_RATE]?.value && !csvDataRow[UTILISATION_REPORT_HEADERS.PAYMENT_CURRENCY]?.value
+  ) {
     return null;
   }
-  if (!validator.isISO4217(paymentCurrencyObject?.value)) {
+  if (!csvDataRow[UTILISATION_REPORT_HEADERS.PAYMENT_CURRENCY]?.value) {
+    return {
+      errorMessage: 'Payment currency must have an entry',
```

Comment:
Seeing as the first `if` block returns `null` if both the exchange rate and currency are missing, I guess this is only required when there is an exchange rate? Would it be worth making that clear in the message here? e.g. something like
```
errorMessage: 'Payment currency must have an entry when an exchange rate is supplied',
```
if so, we could make the same change in the in `generatePaymentExchangeRateError`. e.g.
```
errorMessage: 'Payment exchange rate must have an entry when a payment currency is supplied',
```

Reply:
Yeah I think that is a good idea, I'm just using the copy I'm given but I'll do this and suggest it as a change on the ticket (and revert if I get shot down!)